hydromt_fiat/api/data_types.py

```diff
@@ -13,6 +13,8 @@ class Units(str, Enum):
     meters = "meters"
     feet = "feet"
 
+class Conversion(float, Enum):
+    feet_meters = 3.28084
```

Comment:
This is meters -> feet right?

Reply:
meters > feet and feet> meters, depending whether you multiply or divide

Reply:
I struggled to find a nice name, if you have a nice suggestion

Reply:
I would do this
```
def convert(value: float, old_unit: Unit, new_unit: Unit) -> float:
  if old_unit == new_unit:
    return value
  
  # conversions contains values from meters to any other Unit
  conversions: dict[Unit, float] = {
      Unit.feet: 3.24,
      <other units here>
    }
  if old_unit not in conversions or new_unit not in conversions:
    raise ValueError("Invalid unit")
  
  # convert to meters
  in_meters = value / conversions[old_unit]
  
  # convert to any
  in_any = in_m * conversions[new_unit]
  return in_any
```
which will simplify the code in the other function:
```
if unit != self.unit: 
# this is not required as the convert function handles this case also, but this check will skip looping over the df, so I'd keep it
  self.exposure_db["Ground Elevation"] = self.exposure_db[
    "Ground Elevation"
  ].apply(lambda x: convert(x))
```